spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -135,6 +145,62 @@ If directed graphs are needed instead of the simple linear streams described abo
 == Module Labels
 
 When a stream is comprised of multiple modules with the same name, they must be qualified with labels:
-
+```
   stream create --definition "http | firstLabel: transform --expression=payload.toUpperCase() | secondLabel: transform --expression=payload+'!' | log" --name myStreamWithLabels --deploy
+```
+[[tap-dsl]]
+== Tap DSL
+
+Taps can be created at various producer endpoints in a stream. For a stream like this:
+
+```
+  stream create --definition "http | step1: transform --expression=payload.toUpperCase() | step2: transform --expression=payload+'!' | log" --name mainstream --deploy
+
+```
+taps can be created at the output of `http`, `step1` and `step2`.
+
+To create a stream that acts as a 'tap' on another stream requires to specify the `source destination name` for the tap stream. The syntax for source destination name is:
+
+```
+`<stream-name>.<label/app name>`
```

Comment:
should use a hyphen in `app-name` as well so there's no whitespace... could be confusing